python/ray/serve/mlflow-ray-serve/mlflow_ray_serve/__init__.py

```diff
@@ -0,0 +1,72 @@
+import logging
+
+import ray
+from ray import serve
+from mlflow.deployments import BaseDeploymentClient
+
+import mlflow.pyfunc
+
+logger = logging.getLogger(__name__)
+
+
+def run_local(name, model_uri, flavor=None, config=None):
+    pass  # TODO
+
+
+class MLFlowBackend:
+    def __init__(self, model_uri):
+        self.model = mlflow.pyfunc.load_model(model_uri=model_uri)
+
+    async def __call__(self, request):
+        # TODO: Does this correctly use Ray Serve pandas.df optimization?
+        df = request.data
+        return self.model.predict(df)
```

Comment:
Can we just pass the request bytes in here and mlflow will deserialize?

Reply:
It doesn't look like MLFlow has an API method for reading bytes, only a `predict` method which accepts a pandas dataframe. So deserialization needs to be done outside of MLFlow.